spec/system/pokemon_spec.rb

```diff
@@ -0,0 +1,48 @@
+require "rails_helper"
+
+RSpec.describe "Pokemon", type: :system do
+  before do
+    driven_by(:rack_test)
+  end
+
+  it "enables me to create pokemon" do
+    visit new_pokemon_path
+
+    fill_in "Name", :with => "Groudon"
+    fill_in "Pokemon type", :with => "Boden"
+    fill_in "Region", :with => "Hoenn"
+    click_button "Create Pokemon"
+    
+    expect(page).to have_current_path("/pokemon/1")
+    expect(page).to have_text("Groudon")
+  end
+
+  it "enables me to search for a pokemon" do
+    pokemon_1 = Pokemon.create!(name: "Voltilamm", pokemon_type: "Elektro", region: "Johto")
+    pokemon_2 = Pokemon.create!(name: "Groudon", pokemon_type: "Boden", region: "Hoenn")
+
+    visit root_path
```

Comment:
Visit sehen wir meistens als Teil der Setup Phase an.

```suggestion
    pokemon_1 = Pokemon.create!(name: "Voltilamm", pokemon_type: "Elektro", region: "Johto")
    pokemon_2 = Pokemon.create!(name: "Groudon", pokemon_type: "Boden", region: "Hoenn")
    visit root_path
```